main.go

```diff
@@ -393,14 +393,15 @@ func main() {
 
 		// Temporary solution for non-modularized telemetry operator
 		if !enableTelemetryManagerModule {
-			go func() {
+			go func(certconfig webhookcert.Config, k8sClientCopy client.Client) {
```

Comment:
IMHO, k8sClientCopy is a bit confusing. Maybe let's extract this lambda into a proper function?

Reply:
Done